tests/sysmgr/mount.bats

```diff
@@ -461,70 +473,3 @@ load ../helpers/sysbox-health
 
   rm -rf $mnt_src
 }
-
-@test "bind-mount-uid-shift disable" {
```

Comment:
This test is no longer applicable since the `--disable-uid-shifting` in sysbox-mgr has been deprecated, in favor of `--disable-shiftfs` and `--disable-idmapped-mount`. The test suite now covers the new options extensively.